### ISSUE TITLE
ROFF Product Needs to Meet Cryosphere Requirements

### ISSUE THREAD

**Issue by fastice** [isce-framework/nisar-workflows#10]
As the documentation describes, "The L1_ROFF product is primarily meant for cryosphere applications, and it is only generated for LSAR acquisitions over Antarctica, Greenland, and selected mountain glaciers"

The way the products have been formulated does not meet the cryosphere needs. We have always been adamant that we want the R/D products to be as free from the DEM as possible, especially the offsets. 

The product should be the offset between a point in the reference and the same point in the secondary.  Instead, the documentation says "The pair of L1_RSLC used to produce L1_ROFF is first coarsely aligned with geometrical coregistration using the best available orbit ephemeris and a Digital Elevation Model (DEM) [RD1]." In other words, all of the geometric effects have been removed.

It is fine to compute the offsets this way, but the geometrical coeregistration needs to be added back to offsets so they are as described above.   I understand non-cryosphere groups may have requested them this way, but they should meet the cryosphere requirements. Likewise, they should be in units of pixels, not meters. 

A solution that apparently would make everyone happy is to add to the existing product the raw pixel offsets as described above. The one exception is if a single constant azimuth offset needs to be applied so the offsets are consistent from frame to frame (this just takes out the possibly arbitrary alignment of pairs and is not related to the imaging geometry). These additional fields probably do not need to be added to the GOFF since it has already been heavily affected by the DEM.

**Comment by rignot** [comment]
I agree with Ian. I am ok with meters for the image offsets instead of pixels - even though this was not the original design for Cryosphere and we were not consulted - but I agree with him that we always insisted that we did not want DEM corrections of the image offsets.
